cdap-ui/app/cdap/components/Replicator/Detail/ThroughputGraph/throughput.ts

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import * as d3 from 'd3';
+import numeral from 'numeral';
+import { objectQuery } from 'services/helpers';
+
+export const COLOR_MAP = {
+  inserts: '#185ABC',
+  updates: '#669DF6',
+  deletes: '#AECBFA',
```

Comment:
Where are these colors from?

Reply:
These colors are coming from the design. These does not come from our themes color palette

Reply:
If we are adding colors we might as well add it to the palette.